session_rpcserver.go

```diff
@@ -11,6 +12,21 @@ import (
 	"github.com/lightninglabs/lightning-node-connect/mailbox"
 	"github.com/lightninglabs/lightning-terminal/litrpc"
 	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightningnetwork/lnd/keychain"
+)
+
+var (
+	// TODO(elle): Temp values. change these before landing pr.
+	nBytes, _ = hex.DecodeString(
+		"0254a58cd0f31c008fd0bc9b2dd5ba586144933829f6da33ac4130b555fb5ea32c",
+	)
+	N, _ = btcec.ParsePubKey(nBytes, btcec.S256())
+
+	keyLocator = &keychain.KeyLocator{
```

Comment:
We currently use key family `99` for Loop and `220` for Pool. It doesn't really matter what value we choose. I vote for family 21 and index 0.